daisy_workflows/build-publish/enterprise_linux/almalinux_9.wf.json

```diff
@@ -44,7 +44,8 @@
         "Path": "${workflow_root}/export/disk_export.wf.json",
         "Vars": {
           "destination": "${gcs_url}",
-          "source_disk": "disk-almalinux-9"
+          "source_disk": "disk-almalinux-9",
+          "run-sbom-bool": "true"
```

Comment:
This should be a variable about that defaults to false. Then you can pass in the variable as true for images it should be run on. Then the value here would be the variable.

    "run-sbom-bool": {
      "Value": "false",
      "Description": "Determines if a SBOM should be generated."
    }
    
    Then the value here would be
    "run-sbom-bool": "${run-sbom-bool}"

Reply:
Fixed, running the image build with the same command as before this change should not generate the SBOM.